app/updates/BreakingNewsUpdate.scala

```diff
@@ -136,6 +136,7 @@ class BreakingNewsUpdate(val config: ApplicationConfiguration, val ws: WSClient,
       case Some("uk") => List(BreakingNewsUk)
       case Some("us") => List(BreakingNewsUs)
       case Some("sport") => List(BreakingNewsSport)
+      case Some("election") => List(BreakingNewsElection)
```

Comment:
So in my email that was the unknown part. How does the editions tool receive the "election" topic?
There must be a place where the container is mapped to this string but I fail to see it.

I've also changed that string on the mobile side to be `uk-general-election` just so that we can have other elections in future if necessary.

Reply:
Ah, righto. This comes straight from the client -- on alert send, the client sends a `ClientHydratedCollection` payload which looks like ```
{
  trails: [{
    alert: true
    group: "major"
    headline: "Waves in St Mark's Square as Venice flooded by highest tide in 50 years"
    image: "https://media.guim.co.uk/a09101c5ee771c78ff8d16bdf0f49b5309c34df9/0_131_3072_1842/500.jpg"
    topic: "election"
  }]
}
```

The topic is derived from the `meta.href` property of the collection, which is manually set by the user in the Config tool. If there's not a match, the server sends back an error and the tool tells the user that their alert has failed to send.

Reply:
I'lll change the string accordingly 👍

Reply:
I see, make sense!